images/run-playbook/README.md

```diff
@@ -0,0 +1,43 @@
+# Containerized openshift-ansible to run generic playbooks
+
+This image uses the [playbook2image](https://github.com/aweiteka/playbook2image) source-to-image image to containerize `openshift-ansible`. The resulting image can run any of the provided playbooks.
+
+## Build
+
+1. **Docker**:
```

Comment:
@codificat I'll look at putting the system container files into the playbook2image base so they're inherited for this optional use case. https://github.com/aweiteka/playbook2image/issues/22